<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_usrn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>真实姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_rps">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ps">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_usname">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
